user_docs/en/userGuide.t2t

```diff
@@ -363,10 +363,10 @@ Once you navigate to an object, you can review its content using the [text revie
 When [Focus Highlight #VisionFocusHighlight] is enabled, the location of the current navigator object is also exposed visually.
 By default, the navigator object moves along with the System focus, though this behavior can be toggled on and off.
 
-Note that braille follows both the [focus #SystemFocus] and [caret #SystemCaret] as well as object navigation and text review by default.
-If you want it to follow the focus and caret only, you need to [configure braille to be tethered to #BrailleTether] focus.
-In this case, braille will not follow object navigation and text review.
-If you want braille to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review.
+Note that by default, braille tries to follow automatically both the system focus and system caret as well as the NVDA navigator and the review cursor.
+If you want it to follow the focus and caret only, you need to configure braille to be tethered to focus.
+In this case, braille will not follow the NVDA navigator during object navigation or the review cursor during review.
+If you want braille to follow object navigation and text review instead, you need to configure braille to be tethered to review.

```

Comment:
I think this information belongs in the #BrailleTether section. At the moment it duplicates explanations about settings and defaults. It could be simplified to something like:
```suggestion
Note [braille following object navigation is configurable. #BrailleTether]
```

Then in the #ReviewingText sections:
```suggestion
Note [braille following text review is configurable. #BrailleTether]
```